modules/vaos/app/services/eps/base_service.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Eps
+  class BaseService < Common::Client::Base
+    STS_OAUTH_TOKEN = :va_online_scheduling_sts_oauth_token
+
+    attr_accessor :user
```

Comment:
Is  `attr_accessor :user `necessary? Would  `attr_reader :user `be more appropriate?